alacritty_terminal/src/config/window.rs

```diff
@@ -36,9 +36,17 @@ pub struct WindowConfig {
     #[serde(deserialize_with = "failure_default")]
     pub class: Option<String>,
 
+    /// Window class instance
+    #[serde(deserialize_with = "failure_default")]
+    pub instance: Option<String>,
+
     /// TODO: DEPRECATED
     #[serde(deserialize_with = "failure_default")]
     pub start_maximized: Option<bool>,
+    ///
+    /// Window class instance
+    #[serde(deserialize_with = "failure_default")]
+    pub gtk_theme_variant: Option<String>,
```

Comment:
New entries should be added *above* the deprecated values, to keep them all collected together at the bottom. Also please make sure the whitespace between elements is consistent.